deny.toml

```diff
@@ -0,0 +1,39 @@
+
+# only check for the targets we currently publish
+targets = [
+  { triple = "x86_64-apple-darwin" },
+  { triple = "x86_64-unknown-linux-gnu" },
+  { triple = "x86_64-unknown-linux-musl" },
+  { triple = "x86_64-pc-windows-msvc" },
+]
+
+[advisories]
+vulnerability = "deny"
+unmaintained = "deny"
+notice = "deny"
+unsound = "deny"
+ignore = []
+
+[bans]
+multiple-versions = "deny"
+deny = []
+
+[sources]
+unknown-registry = "deny"
+unknown-git = "deny"
+allow-git = []
+
+[licenses]
+unlicensed = "deny"
+allow-osi-fsf-free = "neither"
+copyleft = "deny"
+confidence-threshold = 0.93
+allow = ["Apache-2.0", "MIT"]
+
+[[licenses.exceptions]]
+allow = ["CC0-1.0"]
```

Comment:
Same also with CC0 in general: we should probably allow that universally.

Reply:
Hmm, without this it warns with

```
warning[L006]: license was not encountered
   ┌─ /home/emil/workspace/cross/deny.toml:31:31
   │
31 │ allow = ["Apache-2.0", "MIT", "CC0-1.0"]
   │                               ^^^^^^^^^ unmatched license allowance

advisories ok, bans ok, licenses ok, sources ok
```

it doesn't make it fail, so I'll change it eitherway

Reply:
This is because it checks each target

Reply:
> This is because it checks each target

Ah ok, maybe then we need something different. I might have to play around with it a bit.